playwright/tests/engine.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from '@playwright/test';
+import { test } from './fixtures/window';
+import { OutputView } from './page-objects/output-view';
+import { launchElectronApp } from './utils/app';
+import { noEngineWorkspacePath, noProjectWorkspacePath } from './workspaces/workspace';
+import { SettingsView } from './page-objects/settings-view';
+import { executeCloseAllEditorGroupsCommand } from './utils/command';
+
+test.describe('Engine Extension', () => {
+  test('check if embedded engine has started', async ({ window }) => {
+    const outputview = new OutputView(window);
+    await outputview.isTabVisible();
+    await outputview.isActive();
+    await outputview.isViewVisible();
+    await outputview.checkIfEngineStarted();
+  });
+
+  test('check default engine settings', async ({}, testInfo) => {
+    const app = await launchElectronApp(noProjectWorkspacePath, testInfo.title);
```

Comment:
not sure about this own test extension...
Doesn't start this test now the ElectronApp twice? Once in the test and once here?

But I assume that you need it right? So playwright doesn't simply start a browser?

But maybe we can add some custom parameters to our own test method so we can directly add the workspace there so it is correctly started right away?

Reply:
> Doesn't start this test now the ElectronApp twice? Once in the test and once here?

no, if `{ window }`is not passed, the test extension does not run. So, it's started once here.

yes, browser does not work :(

> But maybe we can add some custom parameters to our own test method so we can directly add the workspace there so it is correctly started right away?

I tried to do that, but could not find a good solution yet. But I plan to improve it somehow in a followup